src/main/java/com/hillel/items_exchange/service/impl/UserServiceImpl.java

```diff
@@ -93,4 +96,15 @@ public void registerNewUser(UserRegistrationDto userRegistrationDto, BCryptPassw
             log.warn("IN UserServiceImpl (registerNewUser): user: {} not registered", user);
         }
     }
+
+    @Override
+    public Optional<UserDto> getUserDto(Long id) {
+        return userRepository.findById(id).map(this::mapUserToDto);
```

Comment:
`lastOnlineTime` becomes `Null` after that mapping. I think something is wrong.

Reply:
Honestly, I don't know the right answer. The test does show that the field lastOnlineTime outputs with the value null

Reply:
Do you know how to solve such a problem?

Reply:
I think the problem is about types of lastOnlineTine in UserDto and User. They are different, so mapper does something wrong mapping that field

Reply:
That's right, they're different. Then all you have to do is bring them to the same type. I wonder what Roma has to say about that. Thank you for your comment))

Reply:
For fun, I changed the LocalDate type to LocalDateTime in UserDto and now the lastOnlineTime field returns correctly

Reply:
If I were you, I would discuss this issue with Roma and fix it in your PR.

Reply:
@21November do nothing with `last online` for now I'll create ticket to populate this field.